datalad_metalad/extract.py

```diff
@@ -798,8 +798,6 @@ def legacy_get_file_info(dataset: Dataset,
 
 def legacy_extract_file(ea: ExtractionArguments) -> Iterable[dict]:
 
-    import sys
-    print(repr(ea), file=sys.stderr)
```

Comment:
if to be printed -- should be logged IMHO

Reply:
Thanks, that was left in by mistake